model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -206,7 +206,8 @@ case class MethodConfiguration(
                    outputs: Map[String, AttributeString],
                    methodRepoMethod: MethodRepoMethod,
                    methodConfigVersion: Int = 1,
-                   deleted: Boolean = false
+                   deleted: Boolean = false,
```

Comment:
So ... we expose the fact that the method config has been deleted in the API?  We don't do this for Entities / Attributes.  Hm.  This is something we should think about some more.  (not in this PR)